src/schema/v2/image/imageHelper.ts

```diff
@@ -36,8 +31,8 @@ export const hasMissingImageVersion = (image) => {
 export const isProcessingImage = (image) => {
   const geminiTokenUpdatedAt = image.gemini_token_updated_at
 
-  // Return false if missing original or no gemini token update timestamp
-  if (isMissingOriginal(image) || !geminiTokenUpdatedAt) {
```

Comment:
For some reason, `image.image_url` remains `null` for a period after the upload. I have removed it for now. This means that an artwork without an original (e.g., when the uploaded file size is zero) will be displayed as "in progress" for the first 30 minutes and then as "failed." I will merge this fix now and see if we can improve this behavior.